package.json

```diff
@@ -93,7 +93,8 @@
     "start": "craco start",
     "build": "craco build",
     "test": "craco test --testPathPattern=src/web/",
-    "api": "nodemon -r dotenv/config ./src/api/api.ts",
+    "api": "nodemon -e \".ts\" --exec npm run babel-node-api",
```

Comment:
can you explain the actual difference this makes? Does it just ensure that it pulls in the babel config when running?

Reply:
Yes, babel-node-api will use babel-node to transpile the file under api